{{cookiecutter.project_slug}}/Makefile

```diff
@@ -31,6 +31,9 @@ CUSTOM_ENV_NAME?=image:neuromation-{{cookiecutter.project_slug}}
 
 APT_COMMAND?=apt-get -qq
 PIP_COMMAND?=pip -q
+# example:
+# TRAINING_COMMAND='python $(CODE_PATH_ENV)/train.py --log_dir $(RESULTS_PATH_ENV) --data_root $(DATA_PATH_ENV)/cifar10'
```

Comment:
Would you mind to replace this example with a more generic one?

`"bash -c 'cd $(PROJECT_PATH_ENV) && python -u $(CODE_PATH)/train.py --data $(DATA_PATH_ENV)'"`

Reply:
Done